roles/openshift_common/tasks/main.yml

```diff
@@ -4,6 +4,10 @@
   when: openshift_use_openshift_sdn | default(false) | bool and openshift_use_flannel | default(false) | bool
 
 - fail:
+    msg: Nuage can not be used with openshift sdn
+  when: openshift_use_openshift_sdn | default(false) | bool and openshift_use_nuage | default(false) | bool
```

Comment:
There should probably be another test here to ensure that openshift_use_nuage and openshift_use_flannel are both not set to True